auth/user.go

```diff
@@ -207,7 +207,17 @@ func (revokedChannels RevokedChannels) add(chanName string, triggeredBy uint64)
 //  - Revoke the role revoked channels
 // Get user:
 //  - Revoke users revoked channels
-func (user *userImpl) RevokedChannels(since uint64) RevokedChannels {
+func (user *userImpl) RevokedChannels(safeSeq uint64, triggeredBy uint64) RevokedChannels {
+	// We always need to check triggeredBy so that we can validate whether we are resuming an interrupted replication
+	// but we also need to calculate another value to check: checkSeq
+	// checkSeq is either the safeSeq which is passed in, or in the event we have a triggeredBy it may mean we need to
+	// resume an interrupted replication so need to 'step back' one sequence. We do, however, need to check the lowest
+	// of the safeSeq and triggeredBy - 1, in case we have a lowSeq.
+	checkSeq := safeSeq
+	if triggeredBy > 0 && checkSeq > triggeredBy-1 {
```

Comment:
As discussed, the checkSeq > triggeredBy-1 condition is only valid when a low sequence is set.  In the absence of low sequence, we always want to use triggeredBy-1.

Reply:
Addressed